packages/dd-trace/src/config.js

```diff
@@ -270,8 +269,7 @@ class Config {
     }
     const PROPAGATION_STYLE_INJECT = propagationStyle(
       'inject',
-      options.tracePropagationStyle,
-      this._getDefaultPropagationStyle(options)
```

Comment:
Similar concern - this is likely not the correct behavior (even if it equivalent)

@khanayan123 (recent modifier) what is the impact of not including this?

Reply:
This was fixed in a different way later on by @simon-id in https://github.com/DataDog/dd-trace-js/pull/4580